jekyll/_cci2/workflows.adoc

```diff
@@ -279,6 +279,49 @@ curl --request POST \
 ----
 --
 
+
+=== Identify the user who approved a job
+
+[.tab.identify.CircleCI_web_app]
+--
+* If the user signed up via GitHub, the (non-clickable) avatar of the GitHub user appears next to the approval job. Hovering over that avatar brings up a tooltip with the user's name as specified in their GitHub/Bitbucket account settings.
+
+* If the user signed up via email, the anonymous avatar appears next to the approval job; hovering over that avatar brings up a tooltip with the user's email address.
+--
+
+[.tab.identify.API]
+--
+. Set up your API authentication. Steps are available in the xref:api-developers-guide#add-an-api-token[API developers guide].
+. You need the workflow ID so you can use the link:https://circleci.com/docs/api/v2/index.html#operation/listWorkflowJobs[get a workflow's jobs] endpoint to get the user ID.
```

Comment:
It would be good to make these steps even more explicit I think, so a step to show people how to get the workflow ID as step 2, then setep three can be to use that to get the approvers user ID, then a step 4 to get a username and email from that user ID